skimage/__init__.py

```diff
@@ -178,18 +178,6 @@ def _raise_build_error(e):
     from .util.dtype import *
 
 
-def lookfor(what):
-    """Do a keyword search on scikit-image docstrings.
-
-    Parameters
-    ----------
-    what : str
-        Words to look for.
-
-    """
-    import numpy as np
-    import sys
-    return np.lookfor(what, sys.modules[__name__])
-
+from .util import lookfor
```

Comment:
This seems broken to me. Since `util/__init__.py` doesn't import `lookfor`, unless I'm mistaken, here you are importing `lookfor` the module, not `lookfor` the function.

At any rate, it looks like lookfor isn't actually tested anywhere, which to me is a problem!

I suggest a doctest:

```python
>>> import skimage
>>> skimage.lookfor('regular_grid')
Search results for 'regular_grid'
---------------------------------
skimage.util.regular_grid
    Find `n_points` regularly spaced along `ar_shape`.
```

Reply:
hmm sorry, should have been `.uilt.lookfor`